rfc/2019-12-10-add-pulsar-to-binlog.md

```diff
@@ -0,0 +1,40 @@
+# Proposal: Add Apache Pulsar to tidb-binlog
+
+- Author(s): Xiaolong Ran
+- Last updated:  Dec 05, 2019
+- Discussion at: https://github.com/pingcap/community/issues/117
+
+## Abstract
+
+The proposal aims at:
+
+Use **key_shared** subscription type of [Apache Pulsar](https://github.com/apache/pulsar) to enhance **binlog** data processing capabilities;
```

Comment:
The proposal aims at using the **key_shared** subscription type of [Apache Pulsar](https://github.com/apache/pulsar) to enhance **binlog** data processing capabilities.